Exception directives not respected in flake8helled stdin mode

```shell
$ flakehell --version
FlakeHell 0.4.3
Flake8    3.8.1
```

pyproject.toml:

```toml
[tool.flakehell.plugins]
"flake8-*" = ["+*"]

[tool.flakehell.exceptions."test/"]
# don't complain about assert statements in tests, duh
"flake8-bandit" = ["-S101"]
```

test/assertion.py:
```python
assert 2 + 2 == 4
```

flakehell lint run respects the exception:
```shell
$ flakehell lint test/assertion.py
test/assertion.py:1:1: D100 Missing docstring in public module [flake8-docstrings]
```

but flake8helled in stdin mode (which is how vim's lint plugin [ale](https://github.com/dense-analysis/ale/blob/master/ale_linters/python/flake8.vim#L53) runs it) does not:
```shell
$ flake8helled --stdin-display-name test/assertion.py - < test/assertion.py
-:1:1: D100 Missing docstring in public module [flake8-docstrings]
-:1:1: S101 Use of assert detected. The enclosed code will be removed when compiling to optimised byte code. [flake8-bandit]
```

--- a/flakehell/_patched/_checkers.py
+++ b/flakehell/_patched/_checkers.py
@@ -141,7 +141,7 @@ class FlakeHellCheckersManager(Manager):
 
             filename = checker.filename
             if filename is None or filename == '-':
-                filename = self.options.stdin_display_name or "stdin"
+                filename = self.options.stdin_display_name or 'stdin'
             with self.style_guide.processing_file(filename):
                 results_reported += self._handle_results(
                     filename=filename,

--- a/flakehell/_patched/_checkers.py
+++ b/flakehell/_patched/_checkers.py
@@ -139,9 +139,12 @@ class FlakeHellCheckersManager(Manager):
                 results = sorted(checker.results, key=lambda tup: (tup[1], tup[2]))
                 checker.snapshot.dump(results)
 
-            with self.style_guide.processing_file(checker.filename):
+            filename = checker.filename
+            if filename is None or filename == '-':
+                filename = self.options.stdin_display_name or "stdin"
+            with self.style_guide.processing_file(filename):
                 results_reported += self._handle_results(
-                    filename=checker.filename,
+                    filename=filename,
                     results=results,
                     check=checker.check,
                 )

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -1,6 +1,8 @@
 # built-in
 import subprocess
 import sys
+from io import BytesIO
+from unittest.mock import patch, Mock
 from pathlib import Path
 from textwrap import dedent
 
@@ -86,7 +88,7 @@ def test_lint_help(capsys):
     assert '--baseline' in captured.out
 
 
-def test_exclude(capsys, tmp_path: Path):
+def test_exceptions(capsys, tmp_path: Path):
     text = """
     [tool.flakehell.plugins]
     pyflakes = ["+*"]
@@ -109,6 +111,35 @@ def test_exclude(capsys, tmp_path: Path):
     ./tests/test_example.py:2:1: F821 undefined name 'a'
     """
     assert captured.out.strip() == dedent(exp).strip()
+
+
+@patch('sys.stdin.buffer', BytesIO(b'import sys\na\n'))
+@patch('sys.stdin', Mock())
+def test_exceptions_stdin(capsys, tmp_path: Path):
+    # write config
+    text = """
+    [tool.flakehell.plugins]
+    pyflakes = ["+*"]
+
+    [tool.flakehell.exceptions."example.py"]
+    pyflakes = ["-F401"]
+    """
+    (tmp_path / 'pyproject.toml').write_text(dedent(text))
+
+    # make fake `stdin` with source file content
+    # code_path = tmp_path / 'example.py'
+    # code_path.write_text('import sys\na')
+
+    # call the command with passing matching `--stdin-display-name`
+    cmd = ['lint', '--format', 'default', '--stdin-display-name', 'example.py', '--', '-']
+    with chdir(tmp_path):
+        # with code_path.open('r') as stream:
+        result = main(cmd)
+
+    assert result == (1, '')
+    captured = capsys.readouterr()
+    assert captured.err == ''
+    assert captured.out.strip() == "example.py:2:1: F821 undefined name 'a'"
 
 
 def test_baseline(capsys, tmp_path: Path):

